src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  --hover-link-color: #00acdc;
```

Comment:
by the name of the variable, I assume that the color is used only on hover,
but in the code, this variable is also used for the active state
better to rename it to a more abstract name, for example `--accent-color`, `--primary-color`